enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -87,9 +108,27 @@ def _get_enterprise_customer_users_batch(self, start, end):
         Returns a batched queryset of EnterpriseCustomerUser objects.
         """
         LOGGER.info('Fetching new batch of enterprise customer users from indexes: %s to %s', start, end)
-        return User.objects.filter(pk__in=EnterpriseCustomerUser.objects.values('user_id'))[start:end]
+        return User.objects.filter(pk__in=self._get_enterprise_customer_user_ids())[start:end]
 
-    def _assign_enterprise_role_to_users(self, _get_batch_method, options):
+    def _get_enterprise_enrollment_api_admin_users_batch(self, start, end):     # pylint: disable=invalid-name
+        """
+        Returns a batched queryset of User objects.
+        """
+        LOGGER.info('Fetching new batch of enterprise enrollment admin users from indexes: %s to %s', start, end)
+        return User.objects.filter(groups__name=ENTERPRISE_ENROLLMENT_API_ACCESS_GROUP, is_staff=False)[start:end]
+
+    def _get_enterprise_catalog_admin_users_batch(self, start, end):
+        """
+        Returns a batched queryset of User objects.
+        """
+        Application = apps.get_model(OAUTH2_PROVIDER_APPLICATION_MODEL)     # pylint: disable=invalid-name
+        LOGGER.info('Fetching new batch of enterprise catalog admin users from indexes: %s to %s', start, end)
```

Comment:
Why do we need this log message here?

Reply:
To log the information specific to this user role